phylogenetic/defaults/exclude.txt

```diff
@@ -0,0 +1,16 @@
+KAK_428
+NGA/2016/ISTH_0779
+GUINEA_Z_185a
+NIGERIA_IKEJI
+KY425626 # exclude lab passaged
+KY425628 # exclude lab passaged
+KY425632 # exclude lab passaged
+KY425634 # exclude lab passaged
+KY425635 # exclude lab passaged
+KY425638 # exclude lab passaged
+KY425640 # exclude lab passaged
+KY425641 # exclude lab passaged
+KY425642 # exclude lab passaged
+KY425643 # exclude lab passaged
+KY425650 # exclude lab passaged
+KY425651 # exclude lab passaged
```

Comment:
> Someone also mentioned that there may be a way to programmatically exclude lab strains

Super curious how this would be done! If there's a standard way this is flagged in GenBank, I think it'd be helpful for other pathogens as well!

Reply:
> > Someone also mentioned that there may be a way to programmatically exclude lab strains
> 
> Super curious how this would be done! If there's a standard way this is flagged in GenBank, I think it'd be helpful for other pathogens as well!

I was wondering about this too. Spot-checking several of the excluded strains, they all seem to have a `/note` in the `source` feature that includes information about passaging — a first semi-automated approach could just be to flag/warn about strains where the GB file contains "passaged".

Reply:
OH! Just realized that NCBI Datasets includes `is-lab-host` and `lab-host` fields. Maybe those match to the `/lab_host` field in the `source` feature?

Reply:
> Maybe those match to the /lab_host field in the source feature?

Ah they do match! Ran `nextstrain build ingest dump_ncbi_dataset_report` and the results match the accessions added here: 

|Accession   |Is Lab Host  |Lab Host            |
|------------|-------------|--------------------|
|KY425626\.1 |true         |Vero C1008 \(E6\) cells|
|KY425628\.1 |true         |Vero C1008 \(E6\) cells|
|KY425632\.1 |true         |Vero C1008 \(E6\) cells|
|KY425634\.1 |true         |Vero C1008 \(E6\) cells|
|KY425635\.1 |true         |Vero C1008 \(E6\) cells|
|KY425638\.1 |true         |Vero C1008 \(E6\) cells|
|KY425640\.1 |true         |Vero C1008 \(E6\) cells|
|KY425641\.1 |true         |Vero C1008 \(E6\) cells|
|KY425642\.1 |true         |Vero C1008 \(E6\) cells|
|KY425643\.1 |true         |Vero C1008 \(E6\) cells|
|KY425650\.1 |true         |Vero C1008 \(E6\) cells|
|KY425651\.1 |true         |Vero C1008 \(E6\) cells|

Reply:
> > Maybe those match to the /lab_host field in the source feature?
> 
> Ah they do match! Ran `nextstrain build ingest dump_ncbi_dataset_report` and the results match the accessions added here:

How many of the remainder have "is lab host" false?

Reply:
> How many of the remainder have "is lab host" false?

The remaining records just have an empty value, nothing is explicitly marked false.

Reply:
I will probably add lab host fields to ingest in the pathogen-repo-guide ([issue](https://github.com/nextstrain/pathogen-repo-guide/issues/60)), but it doesn't need to block this PR.